http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/BaseCloudServiceRuntimeMonitor.scala

```diff
@@ -103,12 +103,11 @@ abstract class BaseCloudServiceRuntimeMonitor[F[_]] {
   def failedRuntime(monitorContext: MonitorContext,
                     runtimeAndRuntimeConfig: RuntimeAndRuntimeConfig,
                     errorDetails: RuntimeErrorDetails,
-                    instances: Set[DataprocInstance])(
+                    masterInstance: Option[DataprocInstance])(
```

Comment:
I meant to mention in previous PRs that the term `master` has implications that can be painful to various demographics. In the UI we started using `main`. I wonder if we could do that in Leo and elsewhere too.